tests/test_record.py

```diff
@@ -185,3 +199,35 @@ def test_dotted_name(self):
             spider.parse('pass')
             spider.record()
             spider.test()
+
+    def test_skipped_fields(self):
+        with CaseSpider() as spider:
+            spider.imports('import time')
+            spider.custom_settings('''
+                AUTOUNIT_SKIPPED_FIELDS = ['ts']
+            ''')
+            spider.start_requests("yield scrapy.Request('data:text/plain,')")
+            spider.parse('''
+                time.sleep(0.5)
```

Comment:
I think you can just sleep between `spider.record()` and `spider.test()` but this works too :D